mak/SRCS

```diff
@@ -52,8 +52,19 @@ SRCS=\
 	src\core\sync\rwmutex.d \
```

Comment:
Re: commit message: Did you forget to include change to `pthread_np` into the patch?